source/js/modules/Modal.js

```diff
@@ -1,203 +1,230 @@
     }
 
-    /**
+    this.bckg_scrolling = bckg_scrolling;
+    this.sliding = sliding;
+    this._initializeModal();
+  }
+
+  /** Loads a new `ModalContainer` object and updates the `ModalHandler`
      *
      * @param {ModalContainer} modalContainer
      */
-    loadNewContainer(modalContainer) {
-        if (modalContainer) this.container = modalContainer;
-        else throw new Error('[ModalHandler] >>> loadNewContainer() >>> Object reference resulted in a nullptr');
+  loadNewContainer(modalContainer) {
+    if (modalContainer) this.container = modalContainer;
+    else throw new Error('[ModalHandler] >>> loadNewContainer() >>> Object reference resulted in a nullptr');
 
-        this._resetModal();
-        this._initializeModal();
-    }
+    this._resetModal();
+    this._initializeModal();
+  }
 
-    /**
+  /** Loads a new `ModalViewer` object and updates the `ModalHandler`
      *
      * @param {ModalViewer} modalViewer
      */
-    loadNewViewer(modalViewer) {
-        if (modalViewer) this.view = modalViewer;
-        else throw new Error('[ModalHandler] >>> loadNewViewer() >>> Object reference resulted in a nullptr');
+  loadNewViewer(modalViewer) {
+    if (modalViewer) this.view = modalViewer;
+    else throw new Error('[ModalHandler] >>> loadNewViewer() >>> Object reference resulted in a nullptr');
 
-        this._resetModal();
-        this._initializeModal();
-    }
+    this._resetModal();
+    this._initializeModal();
+  }
 
-    _initializeModal() {
-        if (this.container && this.view) {
-            if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML += this.container.HtmlText;
-            else this.view.txtBoxes[0].innerHTML += this.container.HtmlText;
-        } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
-    }
+  _initializeModal() {
+    if (this.container && this.view) {
+      if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML += this.container.HtmlText;
+      else this.view.txtBoxes[0].innerHTML += this.container.HtmlText;
+    } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
+  }
 
-    _resetModal() {
-        if (this.container && this.view) {
-            if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML = '';
-            else {
-                // eslint-disable-next-line no-restricted-syntax
-                for (const txtbox of this.view.txtBoxes) txtbox.innerHTML = '';
-            }
-        } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
-    }
+  _resetModal() {
+    if (this.container && this.view) {
+      if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML = '';
+      else {
+        // eslint-disable-next-line no-restricted-syntax
```

Comment:
isn't it cleaner to just give in to what ESLint is trying to make you do? :)

Reply:
Actually ESLint did nothing just gave me that error, it was a bug since I was iterating through an array and used function on individual html elements while ESLint thought I could use the function on the whole array, which in fact I couldn't.